docker/edge/http/script.sh

```diff
@@ -88,7 +91,58 @@ echo -----------------------------
 echo App ID and Passwords are $ID,$SECRET 
 echo -----------------------------
 
-for connect2 in 1 2 3
+#Ftech PEP PROXY ID 
+for connect3 in 1 2 3
+do 
+   curl --include \
+     --header "X-Auth-token: $token" \
+  "http://$IP_ADDRESS:3000/v1/applications/$ID/pep_proxies" > PEP_Detail.js
+   if [ $? -eq 0 ]; then
+      break
+   else
+      echo reconnecting...
+      continue
+    fi
+done
+
```

Comment:
[Major] what about the scenario when PEP Proxy didn't connected still PEP_Details.js will be read. Need to write the logic.

Reply:
Added code for handling such event